log4j-api/src/test/java/org/apache/logging/log4j/util/LoaderUtilTest.java

```diff
@@ -55,4 +63,53 @@ public void systemClassLoader() {
             thread.setContextClassLoader(tccl);
         }
     }
+
+    @Test
+    public void securityManager() {
+        assertDoesNotThrow(() -> {
+            String separator = System.getProperty("path.separator");
+            List<URL> classpath = new ArrayList<>();
+            for (String entry : System.getProperty("java.class.path").split(separator)) {
+                classpath.add(new File(entry).toURL());
```

Comment:
Looks like this should be `new File(entry).toURI().toURL()` according to the javadoc.